go.mod

```diff
@@ -3,21 +3,30 @@ module github.com/gardener/gardener-resource-manager
 go 1.13
 
 require (
+	github.com/emicklei/go-restful v2.11.0+incompatible // indirect
+	github.com/gardener/external-dns-management v0.0.0-20190722114702-f6b12f6e4b43 // indirect
```

Comment:
Why do we have this? Why are there so many vendor changes?

Reply:
They are coming from "github.com/gardener/gardener/pkg/utils/kubernetes"

Reply:
If you are using gardener/gardener then you should have it in `go.mod`, but I don't see it. Anyways, so far we tried to avoid vendoring gardener/gardener because of follow-up issues with vendoring. Can we try to not vendor it or is it strongly needed?

Reply:
no, Its two lines of code, I need to rewrite it any way to validate truth conditions. Most probably we won't need the extra modules.

Reply:
Currently we are using g/g in the health controller for CheckDaemonSet, CheckDeployment, CheckStatefulSet funcs. Should I revert the g/g vendoring?

Reply:
Hm okay, well, I thought it was only introduced by @zanetworker PR here. If we already had it: OK for now..

Reply:
Gardener dependency was there already, but I used another part that was pruned earlier.